libs/sdk/src/backend/index.ts

```diff
@@ -22,9 +22,9 @@ export interface BackendOptions {
   includeResourceType?: Array<ADCSDK.ResourceType>;
   excludeResourceType?: Array<ADCSDK.ResourceType>;
 
-  cacheKey: string;
-  httpAgent: httpAgent;
-  httpsAgent: httpsAgent;
+  cacheKey?: string;
+  httpAgent?: httpAgent;
+  httpsAgent?: httpsAgent;
```

Comment:
They are mandatory. Every backend must provide them, and any E2E tests containing errors need to be fixed individually.